tests/unit/amo/components/TestGetFirefoxBanner.js

```diff
@@ -93,29 +98,58 @@ describe(__filename, () => {
       expect(root.find(Button).children()).toHaveText('download Firefox');
     });
 
-    it('sets the href on the button with the expected utm params', () => {
+    it('sets the href on the button with the expected utm params for the current variant', () => {
+      const root = render({ store, variant: VARIANT_CURRENT });
+
+      const expectedHref = [
+        `${DOWNLOAD_FIREFOX_BASE_URL}?experiment=${EXPERIMENT_CONFIG.id}`,
+        `variation=${VARIANT_CURRENT}`,
+        `utm_campaign=${DOWNLOAD_FIREFOX_UTM_CAMPAIGN}`,
+        `utm_content=${GET_FIREFOX_BANNER_UTM_CONTENT}`,
+        `utm_medium=referral&utm_source=addons.mozilla.org`,
+        `utm_term=${DOWNLOAD_FIREFOX_UTM_TERM}-${VARIANT_CURRENT}`,
+      ].join('&');
+      expect(root.find(Button)).toHaveProp('href', expectedHref);
+    });
+
+    it('sets the href on the button with the expected utm params for the new variant', () => {
+      const root = render({ store, variant: VARIANT_NEW });
+
+      const expectedHref = [
+        `${DOWNLOAD_FIREFOX_EXPERIMENTAL_URL}?experiment=${EXPERIMENT_CONFIG.id}`,
+        `variation=${VARIANT_NEW}`,
+        `source=amo`,
+        `utm_campaign=${DOWNLOAD_FIREFOX_UTM_CAMPAIGN}`,
+        `utm_content=${GET_FIREFOX_BANNER_UTM_CONTENT}`,
+        `utm_medium=referral&utm_source=addons.mozilla.org`,
+        `utm_term=${DOWNLOAD_FIREFOX_UTM_TERM}-${VARIANT_NEW}`,
+      ].join('&');
+      expect(root.find(Button)).toHaveProp('href', expectedHref);
+    });
+
+    it('sets the href on the button with the expected utm params when there is no variant', () => {
       const root = render({ store });
 
-      const expectedHref = `${DOWNLOAD_FIREFOX_BASE_URL}${makeQueryStringWithUTM(
-        {
-          utm_campaign: DOWNLOAD_FIREFOX_UTM_CAMPAIGN,
-          utm_content: GET_FIREFOX_BANNER_UTM_CONTENT,
-          utm_term: DOWNLOAD_FIREFOX_UTM_TERM,
-        },
-      )}`;
+      const expectedHref = [
+        `${DOWNLOAD_FIREFOX_BASE_URL}?utm_campaign=${DOWNLOAD_FIREFOX_UTM_CAMPAIGN}`,
+        `utm_content=${GET_FIREFOX_BANNER_UTM_CONTENT}`,
+        `utm_medium=referral&utm_source=addons.mozilla.org`,
+        `utm_term=${DOWNLOAD_FIREFOX_UTM_TERM}`,
+      ].join('&');
       expect(root.find(Button)).toHaveProp('href', expectedHref);
     });
 
     it('sends a tracking event when the button is clicked', () => {
       const _tracking = createFakeTracking();
-      const root = render({ _tracking, store });
+      const variant = VARIANT_NEW;
+      const root = render({ _tracking, store, variant });
 
       const event = createFakeEvent();
       root.find(Button).simulate('click', event);
 
       sinon.assert.calledWith(_tracking.sendEvent, {
         action: GET_FIREFOX_BANNER_CLICK_ACTION,
-        category: GET_FIREFOX_BUTTON_CLICK_CATEGORY,
+        category: `${GET_FIREFOX_BUTTON_CLICK_CATEGORY}-${variant}`,
```

Comment:
This simple change broke the GA dashboard on production, which is not nice as we noticed it after a week.. making it hard to work with the data.

Reply:
See: https://github.com/mozilla/addons-frontend/issues/10724